### PR TITLE
set parameters in a better way

### LINKED ISSUE #15: float64 parameters
Using `SCIPSolver("limits/gap",1e-5)` leads to

```
ERROR: LoadError: MethodError: `convert` has no method matching convert(::Type{Ptr{Void}}, ::Float64)
This may have arisen from a call to the constructor Ptr{Void}(...),
since type constructors fall back to convert methods.
Closest candidates are:
  call{T}(::Type{T}, ::Any)
  convert{T}(::Type{Ptr{T}}, !Matched::UInt64)
  convert{T}(::Type{Ptr{T}}, !Matched::Int64)
  ...
 in setparams! at /home/mlubin/.julia/v0.4/SCIP/src/params.jl:13
```

Is this the correct usage? Why do you need to convert a `Float64` to a `void*`?

### PR DESCRIPTION
**Pull request by fserra**
it should fix #15

### CODE REVIEW

File: src/params.jl [hunk 1]
```diff
@@ -7,10 +7,18 @@
...
+function setparameter!(m::SCIPMathProgModel, name::ASCIIString, value::Int)
+    if _getParamType(m, pointer(name)) == 1 # integer
+        return _setIntParam(m, pointer(name), Cint(value))
+    else
+        return _setLongintParam(m, pointer(name), Clonglong(value))
+    end
+end
+setparameter!(m::SCIPMathProgModel, name::ASCIIString, value::Float64) = _setRealParam(m, pointer(name), Cdouble(value))
+setparameter!(m::SCIPMathProgModel, name::ASCIIString, value::Char) = _setCharParam(m, pointer(name), Cchar(value))
+setparameter!(m::SCIPMathProgModel, name::ASCIIString, value::ASCIIString) = _setStringParam(m, pointer(name), pointer(value))
```
Comment (leethargo) [inline src/params.jl]: nice, multiple dispatch :+1:

**Comment by mlubin** [comment]
CSIP 0.3.3 thinks that it's 0.3.2: https://github.com/SCIP-Interfaces/CSIP/blob/3e4a43e0a2f0d9c080ce94683853be628ca802b2/src/csip.c#L10

**Comment by leethargo** [comment]
I know, I just saw it. Trying to move the tag now.

**Comment by leethargo** [comment]
I put a stub related to #14, mostly to retrigger the Travis build.

**Comment by mlubin** [comment]
I'm confused how this is passing since CSIP is still at 0.3.2.

**Comment by leethargo** [comment]
What do you mean with CSIP at 0.3.2?
The build log shows 0.3.3 here: https://travis-ci.org/SCIP-Interfaces/SCIP.jl/jobs/159809451#L715

**Comment by mlubin** [comment]
I mean this: https://github.com/SCIP-Interfaces/CSIP/blob/master/src/csip.c#L10

**Comment by leethargo** [comment]
But this is not the latest commit (and the tag is moved, too).

**Comment by mlubin** [comment]
Maybe a github hiccup.

**Pull request merged**